microsoft/testsuites/gpu/gpusuite.py

```diff
@@ -100,7 +109,7 @@ def verify_gpu_adapter_count(self, node: Node, log_path: Path, log: Logger) -> N
             "Expected device count didn't match Actual device count from lspci",
         ).is_equal_to(expected_count)
 
-        _check_driver_installed(node)
+        ##_check_driver_installed(node)
```

Comment:
Can you also please cleanup the comments?

Reply:
Done.